packages/styling/src/styles/scheme.ts

```diff
@@ -7,31 +7,26 @@ import { ISchemeNames, ITheme } from '../interfaces/index';
  *
  * @param scheme - Scheme to get customizations for.
  * @param context - Context in which to get schemed customizations.
- * @param settingsTheme - Themes setting fallback if context does not have schemed theme.
  */
-export function getSchemedContext(
-  scheme: ISchemeNames,
-  context: ICustomizerContext,
-  settingsTheme?: ITheme
-): ICustomizerContext | undefined {
-  let newContext: ICustomizerContext | undefined;
+export function getSchemedContext(scheme: ISchemeNames, context: ICustomizerContext): ICustomizerContext {
```

Comment:
I'm pretty sure this will get flagged as an API breaking change, but its type signature is backwards compatible (types have been tightened.)